src/nft/components/profile/view/ViewMyNftsAsset.tsx

```diff
@@ -93,7 +93,7 @@ export const ViewMyNftsAsset = ({
 
   const assetMediaType = Card.useAssetMediaType(asset)
 
-  const isDisabled = asset.asset_contract.tokenType === 'ERC1155' || (isSellMode && asset.susFlag)
```

Comment:
we also need to hide floor prices for 1155s not in sell mode

Reply:
sg done.